install/kubernetes/cilium/templates/clustermesh-apiserver/tls-helm/_helpers.tpl

```diff
@@ -0,0 +1,24 @@
+{{/*
+Generate TLS certificates for ClusterMesh.
+
+Note: this line, that is repeated several times below, is a trick to
+ensure the CA certs are generated only once:
+
+    {{- $_ := include "clustermesh-apiserver-generate-certs.helm.setup-ca" . -}}
+
+Please, don't try to "simplify" them as without this trick, every generated
+certificate would be signed by a different CA.
```

Comment:
```suggestion
Note: Always use this template as follows:

    {{- $_ := include "clustermesh-apiserver-generate-certs.helm.setup-ca" . -}}

The assignment to `$_` is required because we store the generated CI in a global `cmca` variable.
Please, don't try to "simplify" this, as without this trick, every generated
certificate would be signed by a different CA.
```